superset/assets/src/visualizations/nvd3_vis.js

```diff
@@ -363,6 +363,13 @@ export default function nvd3Vis(slice, payload) {
         throw new Error('Unrecognized visualization for nvd3' + vizType);
     }
 
+    if (isTruthy(fd.show_brush) && isTruthy(fd.send_time_range)) {
+      chart.focus.dispatch.on('brush', (event) => {
+        const timeRange = event.extent.map(d => d.toISOString().slice(0, -1)).join(' : ');
```

Comment:
This would raise if we ever supported focus on non-time xAxis. Can we make sure that `d` is a date as a condition to trigger the event?